### PR TITLE
Bump typescript version to ~5.5.3

### PR DESCRIPTION
**Pull request by jeremymeng**
and run `rush update`

### CODE REVIEW

File: packages/rlc-common/CHANGELOG.md [hunk 1]
```diff
@@ -43,7 +43,7 @@
 - [Feature] Support additional properties full cases in RLC. Please refer to [#2445](https://github.com/Azure/autorest.typescript/pull/2445).
 - [Feature] Support the change for union and enums in modular and RLC. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
 - [Feature] Upgrade typespec compiler to 0.55.0. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
-- [Feature] Bump typescript version to typescript@~5.4.5. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
+- [Feature] Bump typescript version to typescript@~5.5.3. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
```
Comment (qiaozha) [inline packages/rlc-common/CHANGELOG.md:46]: feel like we don't need to modify the changelog in the past?

File: packages/typespec-ts/CHANGELOG.md [hunk 2]
```diff
@@ -47,7 +47,7 @@
 - [Feature] Support additional properties full cases in RLC. Please refer to [#2445](https://github.com/Azure/autorest.typescript/pull/2445).
 - [Feature] Support the change for union and enums in modular and RLC. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
 - [Feature] Upgrade typespec compiler to 0.55.0. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
-- [Feature] Bump typescript version to typescript@~5.4.5. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
+- [Feature] Bump typescript version to typescript@~5.5.3. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
```
Comment (qiaozha) [inline packages/typespec-ts/CHANGELOG.md:50]: same here.

File: packages/rlc-common/CHANGELOG.md (hunk 1 above)
Comment (jeremymeng) [reply packages/rlc-common/CHANGELOG.md:46]: yes you are right! I was replacing strings and forgot to undo this file

Comment (jeremymeng) [reply packages/rlc-common/CHANGELOG.md]: reverted.

File: packages/typespec-ts/CHANGELOG.md (hunk 2 above)
Comment (jeremymeng) [reply packages/typespec-ts/CHANGELOG.md]: reverted.

**Review by qiaozha — APPROVED** [review]
LGTM

**Pull request merged**